src/Symfony/Component/Clock/MockClock.php

```diff
@@ -20,13 +20,20 @@
  */
 final class MockClock implements ClockInterface
 {
-    private DatePoint $now;
+    private ?DatePoint $now = null;
+    /**
+     * @var (\Closure(): \DateTimeImmutable)|null
+     */
+    private ?\Closure $timeProvider = null;
+    private ?\DateTimeZone $timezone = null;
 
     /**
+     * @param \DateTimeImmutable|string|\Closure(): \DateTimeImmutable $now      The date to return when calling now(), a string that can be parsed into a DateTimeImmutable instance, or a closure that returns a DateTimeImmutable instance
```

Comment:
clearer?
```suggestion
     * @param \DateTimeImmutable|string|(\Closure(): \DateTimeImmutable) $now The date to return when calling now(), a string that can be parsed into a DateTimeImmutable instance, or a closure that returns a DateTimeImmutable instance
```

Reply:
Yes, the Closure return type is the doc can be confusing sometimes